src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,63 @@
+body {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
```

Comment:
place you calendar at the center of the page to avoid cell hiding
<img width="835" alt="Screenshot 2023-11-23 at 20 48 35" src="https://github.com/mate-academy/layout_calendar/assets/117587463/d03466ca-a88b-47c0-8770-45df0d1b54a3">